config_panel.toml

```
@@ -3,6 +3,18 @@ version = "1.0"
 [main]
 name = "Mediawiki configuration"
 
+
+    [main.config]
+    name = "Configuration Options"
+
+        [main.config.create_account]
+        ask = "Create account"
+        type = "boolean"
+        yes = true
+        no = false
+        help = "Set to 'true' to allow account creation."
+        bind = "create_account:__INSTALL_DIR__/LocalSettings.php"
```

```suggestion
        bind = "createaccount:__INSTALL_DIR__/LocalSettings.php"
```

the thing in LocalSettings.php seems to be called without an underscore, but not 100% sure what i'm saying